packages/wallet/src/redux/__tests__/helpers.ts

```diff
@@ -74,51 +79,118 @@ export const expectThisMessage = (state: SideEffectState, messageType: string) =
     expect(item.messagePayload.type).toEqual(messageType);
   });
 };
-export const expectThisMessageAndCommitmentSent = (
-  state: SideEffectState,
-  c: Partial<Commitment>,
-  messageType: string,
-) => {
-  expectSideEffect('messageOutbox', state, item => {
-    expect(item.messagePayload.type).toEqual(messageType);
-    expect(item.messagePayload.signedCommitment.commitment).toMatchObject(c);
-  });
-};
 
-export const expectThisMessageAndTheseCommitmentsSent = (
+type PartialCommitments = Array<{ commitment: Partial<Commitment>; signature?: string }>;
+
+function transformCommitmentToMatcher(sc: { commitment: Partial<Commitment>; signature?: string }) {
+  if (sc.signature) {
+    return expect.objectContaining({ signature: sc.signature });
```

Comment:
Why does the expected object not contain a commitment property?

Reply:
Good catch!